mmcv/ops/nms.py

```diff
@@ -250,7 +264,12 @@ def soft_nms(boxes,
         return dets.to(device=boxes.device), inds.to(device=boxes.device)
 
 
-def batched_nms(boxes, scores, idxs, nms_cfg, class_agnostic=False):
+def batched_nms(
+        boxes: torch.Tensor,
+        scores: torch.Tensor,
+        idxs: torch.Tensor,
+        nms_cfg: Optional[Dict],
```

Comment:
In docsting，`nms_cfg (dict | None): ` -> `nms_cfg (dict, optional): `

Reply:
fixed